package.json

```diff
@@ -29,7 +29,8 @@
     "version": "scripts/release/version.sh",
     "test": "hardhat test",
     "test:inheritance": "node scripts/inheritanceOrdering artifacts/build-info/*",
-    "gas-report": "env ENABLE_GAS_REPORT=true npm run test"
+    "gas-report": "env ENABLE_GAS_REPORT=true npm run test",
+    "slither": "npm run clean && slither . --detect reentrancy-eth,reentrancy-no-eth,reentrancy-unlimited-gas"
```

Comment:
Do you know why `clean` is necessary here?

Reply:
If somebody makes a change to fix the reentrancy or just add the diable comment and doesnt compile before pushing the code, slither will check the last compiled contract and show the warning again, unless you clean it before so slither compiles it itself